applicant/templates/applicant/login.html

```diff
@@ -30,19 +32,17 @@ <h2>Log in to Your Account</h2>
         {% endif %}
     </div>
 
-    
     <div class="divider">
         <h3>
             \
```

Comment:
hafa 'or' frekar ???

Reply:
or is uglier but we could change it back
breytti úr or, gætum lika fundið eh
          /
       or
      /
svg image